include/spdlog/details/spdlog_impl.h

```diff
@@ -14,6 +14,7 @@
 #include <spdlog/sinks/stdout_sinks.h>
 #include <spdlog/sinks/syslog_sink.h>
 #include <spdlog/sinks/ansicolor_sink.h>
+#include <spdlog/sinks/android_sink.h>
```

Comment:
`android_sink.h` has already been guarded with `#if defined(__ANDROID__)`. I think it's redundant to test twice.